language/move-compiler/src/parser/lexer.rs

```diff
@@ -82,6 +82,18 @@ pub enum Tok {
     AtSign,
 }
 
+#[derive(Copy, Clone, Debug, Default)]
+pub struct TextRange {
+    pub start: usize,
+    pub end: usize,
+}
+
+#[derive(Copy, Clone, Debug, Default)]
```

Comment:
Please add documentation to these definitions.

Reply:
done